hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/OzoneFileSystemTests.java

```diff
@@ -95,4 +98,20 @@ private static void listStatusIterator(FileSystem subject,
 
     assertEquals(total, iCount);
   }
+
+  static void createKeyWithECReplicationConfiguration(OzoneConfiguration conf, Path keyPath) throws IOException {
```

Comment:
Let's avoid changing the input config by creating a new one:

```suggestion
  static void createKeyWithECReplicationConfiguration(OzoneConfiguration inputConf, Path keyPath) throws IOException {
    OzoneConfiguration conf = new OzoneConfiguration(inputConf);
  }
```